src/components/donations-actions.tsx

```
@@ -0,0 +1,174 @@
+      alert('There was an error processing your donation. Please try again.');
+    } finally {
+      setLoading(false);
+      setProcessingButton(null);
+    }
+  };
+
+  // Handle custom amount input
+  const handleCustomAmount = (): void => {
+    const amount = prompt('Enter custom amount (USD):');
+    if (amount) {
+      const numAmount = parseInt(amount.replace(/[^0-9]/g, ''));
```

ESLint

[error] Prefer `Number.parseInt` over `parseInt`.

(unicorn/prefer-number-properties)